prow/jobs/kyma/tests/integration/event-service/test-event-service.yaml

```diff
@@ -51,7 +51,7 @@ presubmits: # runs on PRs
       labels:
         <<: *job_labels_template
         preset-build-release: "true"
-    - name: pre-rel12-kyma-tests-gateway-tests
+    - name: pre-rel12-kyma-tests-integration-event-service-tests
```

Comment:
Old releases need their own job template as they don't know the new folder structure

Reply:
Done.